lib/data/member.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:hatspace/data/data.dart';
+
+class Member {
+  final rolesKey = 'roles';
+  final displayNameKey = 'displayName';
+  final Set<Roles> listRoles;
+  final String displayName;
+
+  Member({required this.listRoles, required this.displayName});
+  convertToMap() {
```

Comment:
@dlvinh 
- You should specify data type for code convention, smt like Map<String, dynamic> convertToMap.